acceptance/tests/validate_vendored_ruby.rb

```diff
@@ -177,10 +177,12 @@ def install_dependencies(agent)
     agents_to_test = agents - agents_to_skip
     agents_to_test.each do |agent|
       on(agent, "/opt/puppetlabs/puppet/bin/gem update --system")
```

Comment:
Do we still want to run this? it will take unnecessary time as we skip the expect

Reply:
if the command fails, the test will error out, so at least we check that gem update --system works. See example below:
```
    chance-suburb.delivery.puppetlabs.net (chance-suburb.delivery.puppetlabs.net) executed in 0.78 seconds
    Exited: 1
Beaker::Host::CommandFailure: Host 'chance-suburb.delivery.puppetlabs.net' exited with 1 running:
 /opt/puppetlabs/puppet/bin/gem update --systemaa
Last 10 lines of output were:
	NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
	Gem::Specification#rubyforge_project= called from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/specifications/puppet-5.5.17.gemspec:31.
	NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
	Gem::Specification#rubyforge_project= called from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/specifications/gettext-3.2.2.gemspec:18.
	NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
	Gem::Specification#rubyforge_project= called from /opt/puppetlabs/puppet/lib/ruby/vendor_gems/specifications/highline-1.6.21.gemspec:19.
	NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
	Gem::Specification#rubyforge_project= called from /opt/puppetlabs/puppet/lib/ruby/gems/2.4.0/specifications/text-1.3.1.gemspec:18.
	ERROR:  While executing gem ... (OptionParser::InvalidOption)
	    invalid option: --systemaa
```